assets/js/main.js

```diff
@@ -78,3 +78,23 @@ $(".show-hide").click(function() {
     $(this).removeClass("ms-icon icon-hide").addClass("ms-icon icon-view");
   }
 });
+
+// navigation partials
+
+$('.dpsg-link').on('click', function() {
+  var partial = $(this).attr('data-partial'),
+    elementClicked = this;
```

Comment:
Wouldn't it be best to avoid having an extra variable here? `elementClicked` You could use `this` below